### PR TITLE
Navigation fix for process tab and workflow tab in Guide page

### PR DESCRIPTION
**Pull request by meghanakr7**
Fix for [#471](https://github.com/ESIPFed/Geoweaver/issues/471)

### CODE REVIEW

**Comment by gokulprathin8** [comment]
@meghanakr7  the `New Process` and `Add to Weaver` styled buttons on the same `geoweaver.html` file have duplicate ids on them. This can cause weird behavior
we either need to remove the ids or make the buttons trigger the dialog.

File: src/main/resources/templates/geoweaver.html [hunk 1]
```diff
@@ -364,7 +364,7 @@
 						
 					</div>
 
-					<div id="right-page-div" class="row" style="height:calc( 100% - 50px ); left:0; margin:0; padding: 0;padding-bottom:0px;">
+					<div id="right-page-div" class="row" style="height:100%; left:0; margin:0; padding: 0;padding-bottom:0px;">
```
Comment (ZihengSun) [inline src/main/resources/templates/geoweaver.html:367]: Cannot scroll to the bottom. The -50px is to ensure all the page content can be seen.

**Review by ZihengSun — CHANGES_REQUESTED** [review]
One comment

**Comment by meghanakr7** [comment]
> One comment

Reverted change related to height.

File: src/main/resources/templates/geoweaver.html [hunk 2]
```diff
@@ -168,12 +168,12 @@
...
 		<div id="jsframe-container"></div>
 	
-		<div class="container" style="margin:0; padding:0; width:100%; height:calc(100vh);">
+		<div class="container" style="margin:0; padding:0; width:100%; height:calc(100vh); display: flex;">
 			
 			<!-- Top menu bar -->
 
 			<!-- set site margin here -->
-			<div class="row" id="site_main_div" style="height:calc(100vh); top: 80px; margin: 0px; padding: 0px; padding-top: 1px;">
+			<div class="row" id="site_main_div" style="height:calc(100vh); top: 80px; margin: 0px; padding: 0px; padding-top: 1px; width:100%; flex-shrink: 0; flex-basis: auto;">
```
Comment (gokulprathin8) [inline src/main/resources/templates/geoweaver.html:176]: Clicking on process / workflow hides the top navigation bar. Please recheck css

**Review by gokulprathin8 — CHANGES_REQUESTED** [review]

**Comment by gokulprathin8** [comment]
this fixes the view port to display content at 100% vh, please open a different PR to address the issue of fixing top nav being hidden for `top_nav` div

@meghanakr7

**Review by gokulprathin8 — APPROVED** [review]

**Pull request merged**